declaration/AxisManager.d.ts

```diff
@@ -0,0 +1,26 @@
+import { AxesOption } from "./Axes";
+export interface Axis {
+    [key: string]: number;
+}
+export interface AxisOption {
```

Comment:
@sculove I thought AxesOption and AxisOption are the same. It's my mistake. Sorry.